TemporaryOysters/bangazon_website
# Zach

##Descriptions:

Created PaymentType Model and Tests that go with it. Merged In Customer and Cleaned up Tests to run with Customer. 

This Pr will allow us to move forward where we have foreign keys from paymenttype.

All Code is simple with documentation.


I am working towards completing the feature #8 

##Steps to Test:
Go to master
Fetch branch 'zach'
Pull down 'zach' branch
In command line
run: python manage.py makemigrations
python manage.py migrate
python manage.py test bangazon_webstore

9 total tests should pass OK. 
Congrats I am the Man!

##Related Issue:

#8, and #12
